src/main/java/com/mparticle/kits/ButtonKit.java

```diff
@@ -73,148 +90,180 @@ public ButtonKit getInstance() {
     }
 
     @Override
-    public void checkForDeepLink() {
-        final DeepLinkListener onLink = new DeepLinkListener() {
-            @Override
-            public void onDeepLink(final Intent deepLinkIntent) {
-                    DeepLinkResult result = new DeepLinkResult()
-                            .setLink(deepLinkIntent.getDataString())
-                            .setServiceProviderId(getConfiguration().getKitId());
-                    getKitManager().onResult(result);
-            }
+    public void setInstallReferrer(Intent intent) {
+        if (intent != null) {
+            logDebug("Tracking incoming intent from %s", intent.toString());
+            ButtonMerchant.trackIncomingIntent(getApplicationContext(), intent);
```

Comment:
Install referrer intent comes from the play store and does not contain the attribution token. This functionality has been removed and no longer needs to be tracked.